includes/sanitizers/class-amp-iframe-sanitizer.php

```diff
@@ -192,4 +192,28 @@ private function build_placeholder( $parent_attributes ) {
 
 		return $placeholder_node;
 	}
+
+	/**
+	 * Removes a child of a node.
+	 *
+	 * Also, calls the mutation callback for it.
+	 * This tracks all the nodes that were removed.
+	 *
+	 * @since 0.7
+	 *
+	 * @param object $child The node to remove.
+	 * @param object $parent The parent node for which to remove the child (optional).
+	 * @return void.
+	 */
+	public function remove_child( $child, $parent = null ) {
+		if ( ( null === $parent ) && method_exists( $child->parentNode, 'removeChild' ) ) { // phpcs:ignore WordPress.NamingConventions.ValidVariableName.NotSnakeCaseMemberVar.
+			$child->parentNode->removeChild( $child ); // phpcs:ignore WordPress.NamingConventions.ValidVariableName.NotSnakeCaseMemberVar.
+		} elseif ( method_exists( $parent, 'removeChild' ) ) {
+			$parent->removeChild( $child );
+		}
+
+		if ( isset( $this->args['mutation_callback'] ) ) {
```

Comment:
It's possible that `remove_child` could be called without any node being removed, per the above `if`/`elseif` conditions. Should this check to see if something was actually removed? I'm curious as to the scenarios when the `else` condition would be met.

Reply:
Thanks, that's a good point. This [commit](https://github.com/Automattic/amp-wp/pull/912/commits/90090b0727cd652c197be8391bb21f8d72614793) from above has the `if` conditional nested, to ensure there was a removal before calling the `'mutation_callback'`.